src/bindings/python/src/pyopenvino/graph/node.cpp

```diff
@@ -216,6 +217,58 @@ void regclass_graph_Node(py::module m) {
                 :return: Number of inputs.
                 :rtype: int
              )");
+    node.def("get_input_element_type",
+             &ov::Node::get_input_element_type,
+             py::arg("index"),
+             R"(
+                Returns the element type for input index
+
+                :param index: Index of the input.
+                :type index: int
+                :return: Type of the input index
+                :rtype: openvino.runtime.Type
+             )");
+    node.def("get_input_partial_shape",
+             &ov::Node::get_input_partial_shape,
+             py::arg("index"),
+             R"(
+                Returns the partial shape for input index
+
+                :param index: Index of the input.
+                :type index: int
+                :return: PartialShape of the input index
+                :rtype: openvino.runtime.PartialShape
+             )");
+    node.def("get_input_shape",
+             &ov::Node::get_input_shape,
+             py::arg("index"),
+             R"(
+                Returns the shape for input index
+
+                :param index: Index of the input.
+                :return: Shape of the input index
+                :rtype: openvino.runtime.Shape
+             )");
+    node.def("set_output_type",
+             &ov::Node::set_output_type,
+             py::arg("index"),
+             py::arg("element_type"),
+             py::arg("shape"),
+             R"(
+                Sets output's element type and shape.
+
+                :param index: Index of the output.
+                :param element_type: Element type of the output.
+                :param shape: Shape of the output.
```

Comment:
we can also add types for each input. For other methods, they have

Reply:
done